### PR TITLE
Add `aframe-init` CLI tool for initializing directory with config files to launch pipeline

### PR DESCRIPTION
**Pull request by EthanMarx**
Adds `aframe-init` CLI tool for initializing directory with config files to launch pipeline. As an example,

```bash
poetry run aframe-init sandbox --directory /home/ethan.marx/aframe/analysis/
```

Will copy the sandbox `base.cfg`, `bbh.cfg` and training `config.yaml` to `/home/ethan.marx/aframe/analysis/`. It will also populate a `env.env` with the appropriate environment variable paths corresponding to the `--directory` you passed.

You can also optionally specify an `--s3-bucket`

```bash
poetry run aframe-init sandbox --directory /home/ethan.marx/aframe/analysis/ --s3-bucket s3://aframe-test/test/
```

This will add set the `AFRAME_TRAIN_RUN_DIR` and `AFRAME_TRAIN_DATA_DIR` to the s3 bucket location, which will initialize remote training

To initialize a `tune` pipeline, just run 

```bash
poetry run aframe-init tune --directory /home/ethan.marx/aframe/tune/
```

This will also copy over a search_space.py file for editing.

### CODE REVIEW

**Comment by wbenoit26** [comment]
Can you also add an option for the BNS config? Not sure what the neatest way to do that will be, but it would be nice to be able to do either

**Comment by wbenoit26** [comment]
I like this concept. How do you envision the workflow in this scenario? Do you still launch the pipeline from the root of the repo?

**Comment by EthanMarx** [comment]
I think for now if you're doing `bns` you'll have to just adjust the parameters yourself

**Comment by EthanMarx** [comment]
> I like this concept. How do you envision the workflow in this scenario? Do you still launch the pipeline from the root of the repo?

Yeah I think you'll still have to launch the pipeline from the root with poetry. Maybe I can add an extra file that gives you the corresponding command to copy

**Review by wbenoit26 — APPROVED** [review]

**Comment by EthanMarx** [comment]
@wbenoit26 Now made the `.env` a runfile that adds the poetry command for launching the pipeline (which can be run from anywhere, taking advantage of the poetry `--directory` arg to run poetry from a specified location). Users now have to add, yet another, environment variable `$AFRAME_REPO` to tell us where the aframe code is located. 

Now that we have this and the `build-containers` executable it's much simpler to get started. 
Going to update the top level `README` to have new users take advantage of these executables. If they wan't more information on what's going on, can point them to the various sub readmes

File: scripts/aframe_init.py [hunk 1]
```diff
@@ -74,24 +76,42 @@ def copy_configs(
...
+    path: Path, pipeline: str, s3_bucket: Optional[Path] = None
+):
     # if s3 bucket is provided
     # store training data and training info there
     base = path if s3_bucket is None else s3_bucket
+
+    config = path / f"{pipeline}.cfg"
+    # make the below one string
+    cmd = f"LAW_CONFIG_FILE={config} poetry run --directory $AFRAME_REPO "
+    cmd += f"law runaframe.pipelines.{pipeline}.{pipeline.capitalize()} "
```
[the remark this answers is not in the document]
Comment (EthanMarx) [reply scripts/aframe_init.py:89]: good eye

File: scripts/aframe_init.py [hunk 2]
```diff
@@ -74,24 +76,42 @@ def copy_configs(
...
+def create_runfile(
+    path: Path, pipeline: str, s3_bucket: Optional[Path] = None
+):
     # if s3 bucket is provided
     # store training data and training info there
     base = path if s3_bucket is None else s3_bucket
+
+    config = path / f"{pipeline}.cfg"
+    # make the below one string
+    cmd = f"LAW_CONFIG_FILE={config} poetry run --directory $AFRAME_REPO "
```
[the remark this answers is not in the document]
Comment (EthanMarx) [reply scripts/aframe_init.py:88]: Good call. Makes things simpler

**Comment by EthanMarx** [comment]
@wbenoit26 Updated the README to use these new executables. Should be much simpler to get going now. When you get a chance give those a run through

File: aframe/pipelines/sandbox/README.md [hunk 3]
```diff
@@ -1,33 +1,43 @@
-# Sandbox Pipeline
+# Pipelines
 
 > **_Note: It is highly recommended that you have completed the [ml4gw quickstart](https://github.com/ml4gw/quickstart/) instructions before running the sandbox pipeline_**
 
 > **_Note: It is assumed that you have already built each [project's container](../../../projects/README.md)_**
 
-The sandbox pipeline strings together `luigi` / `law` tasks to run an end-to-end generic aframe workflow.
+Pipelines string together `luigi` / `law` tasks to run an end-to-end generic aframe workflow.
 In short, running the sandbox pipeline will
```
Comment (wbenoit26) [inline aframe/pipelines/sandbox/README.md:8]: Maybe "In short, these pipelines can:"

File: README.md [hunk 4]
```diff
@@ -38,15 +38,35 @@ git submodule update --init
...
+
+```bash
 poetry install
 ```
-This environment is used to launch `luigi`/`law` tasks and pipelines. See this [README](./aframe) for more information.
 
-Next, follow the [instructions](./projects/README.md) for building each project's Apptainer images, and familiarize yourself with the various projects. These images are used as environments for running `Aframe` workflows, and thus are necessary to build. Once complete, you are all setup! 
+This environment is used to launch `luigi`/`law` tasks and pipelines (See this [README](./aframe) for more information), and also contains other helpful command line utilities for building project container images, and initializing directories with configuration files for various analyses. 
+
+You can now build each of the project apptainer container images. Set the `AFRAME_CONTAINER_ROOT` environment variable where the image files will be stored. We recommend Something like `~/aframe/images`. This might take ~ 10 minutes, so grab a coffee!
```
Comment (wbenoit26) [inline README.md:48]: "Something" is accidentally capitalized

File: README.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ Once setup, create a [fork](https://docs.github.com/en/pull-requests/collaborati
 git clone git@github.com:albert-einstein/aframev2.git
 ```
 
-Set the `AFRAME_REPO` environment variable in your `~/.bash_profile` to point to the location where you cloned the repository.
+Set the `AFRAME_REPO` environment variable in your `~/.bash_profile` to point to the location where you cloned the repository.
```
Comment (wbenoit26) [inline README.md:30]: Had a couple students confused about what this meant, just from not being familiar with the shell. Could you add the explicit command to add the variable to the file, and same with `AFRAME_CONTAINER_ROOT` below?

Comment (wbenoit26) [reply README.md:30]: And also an instruction to `source ~/.bash_profile`

**Review by wbenoit26 — APPROVED** [review]
